upstream-community-operators/falco/falco-operator.v0.5.6.clusterserviceversion.yaml

```diff
@@ -0,0 +1,95 @@
+    mediatype: image/svg+xml
+  install:
+    spec:
+      deployments:
+      - name: falco-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: falco-operator
+          strategy: {}
+          template:
+            metadata:
+              creationTimestamp: null
+              labels:
+                name: falco-operator
+            spec:
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: OPERATOR_NAME
+                  value: falco-operator
+                image: docker.io/falcosecurity/falco-operator-helm:0.5.6
+                imagePullPolicy: Always
+                name: falco-operator
+                ports:
+                - containerPort: 60000
+                  name: metrics
+                resources: {}
+              serviceAccountName: falco-operator
+    strategy: deployment
+  maturity: Basic Install
```

Comment:
This should be removed.

Reply:
I'm not sure about the entry/entries that should be removed. Are you referring only to the maturity entry?